packages/tools/fozzie/src/scss/settings/_variables.scss

```
@@ -11,47 +11,47 @@
 // Set this in pixels (but do not add px),
 // the font-size mixin will convert to REMS
 
-$font-size-base             : nth(dt.$font-size-c, 1); // Base font size is set to 16 (`body-l` size)
+$font-size-base             : nth(dt.$font-size-16, 1); // Base font size is set to 16 (`body-l` size)
 $font-size-base-px          : $font-size-base * 1px;
 
 // Type map – these are our global font-sizes and line-heights across the site
 $type: (
     caption: (
-        default: (dt.$font-size-a)
+        default: (dt.$font-size-12, dt.$font-size-12-line-height)
```

Super small NIT. Could you format this like the below for these entries? Think it makes it a bit more readable:

```suggestion
        default: (
            dt.$font-size-12,
            dt.$font-size-12-line-height
        )
```